src/txmempool.h

```diff
@@ -152,6 +153,15 @@ class CTxMemPool
 
     mutable CCriticalSection cs;
     indexed_transaction_set mapTx;
+
+private:
+    typedef std::map<CMempoolAddressDeltaKey, CMempoolAddressDelta, CMempoolAddressDeltaKeyCompare> addressDeltaMap;
```

Comment:
Do we end up using these typedefs more later? I personally prefer having the raw types. It's convenient not to have to type all this when we use these in `txmempool.cpp`, but in those cases it looks like we can get around that with the `auto` keyword.

Reply:
@Eirik0, good suggestion, fixed in 18d9594ce7106403abf6119fb204064acbe05c15. I also improved a bunch of other stuff (force-pushed), so please take at least a quick look at everything again.